demo.js

```diff
@@ -14,6 +14,40 @@ var caf = window.cancelRequestAnimationFrame ||
 
 
 var gamepads = new window.Gamepads();
+
+gamepads.setIndices({
+  'standard': {
+    cursorX: 2,
```

Comment:
it's ambiguous whether this is a button or axis. for now, this shouldn't matter - but this won't necessarily scale.